packages/@okta/vuepress-site/docs/concepts/event-hooks/index.md

```diff
@@ -125,7 +128,9 @@ After implementing your external service, you need to register it with Okta. To
 
 After registering the Event Hook, you need to trigger a one-time verification process by clicking the Verify button that is displayed in Admin Console. When you trigger verification, Okta calls out to your external service, making the one-time verification request to it. You need to have implemented functionality in your service to handle the expected request and response. The purpose of this step is to prove that you control the endpoint. See [One-Time Verification Request](/docs/concepts/event-hooks/#one-time-verification-request) for more information.
 
-For a working example of an end-to-end Event Hook set up, see the following guide [Event Hooks](/docs/guides/event-hook-implementation).
+### Preview your hook
+
+You can preview the the JSON payload for the Event Hook request from the Admin Console's **Preview** tab, which provides a review of the Event Hook request syntax for the specific Event Type. If verified, the request can be delivered to your external service to make sure it is successfully received. See [Event Hook Preview](https://help.okta.com/en/prod/Content/Topics/automation-hooks/event-hooks-preview.htm) for more information.
```

Comment:
@brianduffield-okta The Preview feature actually works for event hooks in both verified and unverified state to reduce friction for developers to test Event Hooks.  I would remove the "If verified," clause.